### ISSUE TITLE
Changing look&feel does not work on Linux

### ISSUE THREAD

**Issue by lehnerpat** [smetzlaff/openDLX#6]
On my Linux (Ubuntu 13.04 x64) with a current Java, I cannot change the look&feel of the simulator GUI via the items in the `Window` menu.

I'll try to debug this a bit when I get the time. If someone else could counter-check this, possibly on other distributions, it would help to see how localized this problem is.

PS: In this case, "does not work" means "nothing happens at all". The "System L&F" radiobutton is activated after clicking it, but nothing happens to the UI. After restarting the simulator, the Java L&F is again selected in the menu.

PPS: Changing the L&F works on Windows.

**Comment by smetzlaff** [comment]
I've never experienced any change of the look&feel at my Debian Wheezy. I think this only useful for Windows or MacOS. I have no preference to keep this option. But since most of the users have Windows, a better solution would be to hide this option only for Linux.

**Comment by lehnerpat** [comment]
I just investigated further and found out that it is in fact not broken on Linux. My Java Environment just returns the same L&F as "System" and "Java" variants (in both cases it's the "Metal" style, which is the Swing default).

However, it is relatively easy to get a list of available L&Fs from the `UIManager`. Therefore, it would make sense to make the menu dynamic and offer all L&Fs it knows.

I'll work out a patch for that and open a pull request once the first commit for it is done. Things to do include:
- create menu items at UI initialization, one for each L&F
- modify `LookAndFeelStrategy` class and subclasses to dynamically accept an L&F class name to activate
- bind the menu items to trigger the L&F activation using that modified class

Closing this issue for now, since it is not actually "broken".
